web-src/src/common/components/history/executions-log-table.vue

```diff
@@ -44,38 +62,64 @@ export default {
     }
   },
 
+  data() {
+    return {
+      filteredRows: this.rows ? [...this.rows] : [],
+      searchText: '',
+      mySortColumn: this.sortColumn,
+      myAscending: this.ascending
```

Comment:
Why do we need those 2? I think properties should be enough

Reply:
The reason for this change is - since we are changing the value of these properties, vue is giving warnings. Attaching the warning screen shots
<img width="1419" alt="Screenshot 2022-09-21 at 6 18 44 PM" src="https://user-images.githubusercontent.com/1894828/193419705-ffb31608-6880-4777-ac45-f73c355a5d6d.png">
<img width="1430" alt="Screenshot 2022-09-21 at 6 18 52 PM" src="https://user-images.githubusercontent.com/1894828/193419709-21c0e22c-8030-4574-8aa2-6a4e7c4f6ac5.png">

Reply:
Oh, I see, thanks. I think it was a mistake in the previous PR. ascending and sortColumn should be defined as data, not props. Could you remove them from props, please and just use `ascending` and `sortColumn` names in data